### DISCUSSION TITLE
Heads up: interesting Rust MSRV discussions

### DISCUSSION THREAD

**Discussion by Kixunil** [rust-bitcoin/rust-bitcoin#1142] [General]
Just want to bring to attention MSRV discussion in the Rust community. Since Bitcoin libraries have a quite old MSRV it may be interesting for people here. Especially @TheBlueMatt who I saw being involved in Debian discussions.

[MSRV policy for the libc crate](https://github.com/rust-lang/libs-team/issues/72)
[Rust MSRV policy vs Linux Distros ](https://internals.rust-lang.org/t/rust-msrv-policy-vs-linux-distros/17074)

**Comment by apoelstra** [comment]
Thank you for engaging politely. Several particularly loud people in that thread I have long since given up on. I am not interested in engaging further myself.

Not surprised in the least that they are spreading this nonsense to *libc* of all crates.

Was happy to learn about Ferrocene. Getting ISO certifications is similar to our goals of getting a trusted compiler toolchain. May be worth keeping an eye on. Funny that the thread had exactly one mention of toolchain trust in 150 comments. I think that confirms the speculation that "people who care about this sort of thing have already given up on Rust".

**Reply by TheBlueMatt** [reply]
Yea, I posted something about x-lang LTO cause its something they kinda care about, "toolchain trust" in the Rust ecosystem, sadly, has become like talking about bigfoot.

**Reply by Kixunil** [reply]
Interesting, I don't perceive most people in the thread as too bad yet but indeed, it does surprise me that so few people care about toolchain trust.
